proto/spire/api/types/bundle.proto

```diff
@@ -18,14 +18,20 @@ message Bundle {
 
     // The sequence number of the bundle.
     uint64 sequence_number = 5;
+
+    // The list of X.509 authorities that are no longer secure to use, 
+    // and must be rotated.
+    repeated X509TaintedKey x509_tainted_keys = 6;
```

Comment:
It is only required for x509 authorities, but it feels strange to not have the same list for JWT authorities, what do you think?